virttest/qemu_devices/qdevices.py

```diff
@@ -993,6 +994,109 @@ def verify_unplug(self, dev_type, monitor):
         return True
 
 
+@total_ordering
+class IOThread(QObject):
+    """iothread object.
+
+    attributes of iothread
+    ["id", "poll-max-ns", "poll-grow", "poll-shrink"]
+    """
+
+    def __init__(self, id, params=None):
+        """Constructor."""
+        if params is None:
+            params = dict()
+        params["id"] = id
+        self.id = id
+        self.__attached_devices = list()
+        kwargs = dict(backend="iothread", params=params)
+        super(IOThread, self).__init__(**kwargs)
+
+    def __len__(self):
+        """Get count of attached devices."""
```

Comment:
The definition is conflict with the one of `QBaseDevice`, so should use another name instead.